velox/exec/fuzzer/WindowFuzzer.cpp

```diff
@@ -89,19 +91,32 @@ std::tuple<std::string, bool> WindowFuzzer::generateFrameClause() {
   }
 
   auto frameBound =
-      [minKValue, maxKValue, this](
+      [this, minKValue, maxKValue, &argNames, &argTypes](
           core::WindowNode::BoundType boundType) -> const std::string {
-    // Generating only constant bounded k PRECEDING/FOLLOWING frames for now.
     auto kValue = boost::random::uniform_int_distribution<int64_t>(
         minKValue, maxKValue)(rng_);
     switch (boundType) {
       case core::WindowNode::BoundType::kUnboundedPreceding:
         return "UNBOUNDED PRECEDING";
       case core::WindowNode::BoundType::kPreceding:
+        if (vectorFuzzer_.coinToss(0.5)) {
+          // Generating column bounded k PRECEDING frames which are positive
```

Comment:
Rephrase :
"Generating column bounded frames 50% of the time and constant bounded the rest of the times. The column bounded frames are of type INTEGER and only have positive values as Window functions reject negative values for them"

Same wording in both places.